app/src/main/java/org/puder/trs80/MainActivity.java

```diff
@@ -417,7 +417,7 @@ private void showCommunity() {
 
     private void downloadROMs() {
         AlertDialog.Builder builder = AlertDialogUtil.createAlertDialog(this,
-                R.string.title_initial_setup, R.drawable.warning_icon, R.string.initial_setup);
+                R.string.title_initial_setup, R.drawable.ic_info_black_24px, R.string.initial_setup);
```

Comment:
Do you know that you don't have to download anything anymore? This is part of Android Studio now. Just go to File > New > Vector Assets. This will pop open a wizard that lets you easily import vector assets. This wizard will use the _dp suffix (I'm surprised that what you downloaded is in px). BTW: I have given my icon assets a 80% opacity (which you can also easily set in the vector asset wizard)